backend/dissemination/models.py

```diff
@@ -505,3 +506,19 @@ class SecondaryAuditor(models.Model):
     report_id = models.TextField(
         REPORT_ID_FK_HELP_TEXT,
     )
+
+
+class MigrationChangeRecord(models.Model):
+    audit_year = models.TextField(blank=True, null=True)
+    dbkey = models.TextField(blank=True, null=True)
+    sac = models.ForeignKey(
```

Comment:
As discussed during the dev huddle, it seems that there is no consensus yet on what to do with the sac objects generated by historic data migration (we will most likely figure this very soon). In this event, the agreement is to delete these sac objects or move them elsewhere (like into the historic data database), we will need to change `sac` to `report_id` here and remove the ForeignKey. TBD

Reply:
Based on Matt's note in the slack channel, sac will be replaced with record_id.